formats/opentype.fathom

```diff
@@ -291,15 +291,15 @@ def table_record = {
 
 /// Find a table record using the given `table_id`.
 def find_table =
-    fun (num_tables : U16) =>
+    fun (@num_tables : U16) =>
     fun (table_records : Array16 num_tables (Repr table_record)) =>
     fun (table_id : Repr tag) =>
         // TODO: accelerate using binary search
         // TODO: make use of `table_record.search_range`
         // TODO: make use of `table_record.entry_selector`
         // TODO: make use of `table_record.range_shift`
-        array16_find _ (Repr table_record)
-            (fun table_record => table_record.table_id == table_id)
+        array16_find
+            (fun (table_record : Repr table_record) => table_record.table_id == table_id)
             table_records;
```

Comment:
Not sure if postponement would help with avoiding the need for a type annotation here? The issue is that we don’t know how to elaborate `table_record.table_id` without knowing the type of `table_record` parameter before hand which comes from the `table_records` argument that is supplied later.